### PR TITLE
Refactor dashboard/feed.tmpl

### PR DESCRIPTION
**Pull request by CaiCandong**
- ~~Refactor `ActionType` to `models/activities/action_type.go`~~
- Replace the magic number in `feed.tmlp` with `InAction`

### CODE REVIEW

**Review by lunny — APPROVED** [review]

File: templates/user/dashboard/feeds.tmpl [hunk 1]
```diff
@@ -11,75 +11,75 @@
 					{{else}}
 						{{.ShortActUserName}}
 					{{end}}
-					{{if eq .GetOpType 1}}
+					{{if .GetOpType.IsActionCreateRepo}}
```
Comment (wxiaoguang) [inline templates/user/dashboard/feeds.tmpl]: What's the difference of `eq .GetOpType.String "create_repo"` or `.GetOpType.IsAction "create_repo"` ?

IMO these `IsXxx` functions only increase code lines, but doesn't bring real benefit. eg: the code is still executed dynamically and no static check.

Comment (CaiCandong) [reply templates/user/dashboard/feeds.tmpl]: I would like to improve the readability of the code in tmpl, `.GetOpType.IsAction "create_repo"` looks more convenient.

Comment (wxiaoguang) [reply templates/user/dashboard/feeds.tmpl]: I guess it can be `func IsAction(actions ...string) bool`, then the template `or` can also be eliminated

Comment (CaiCandong) [reply templates/user/dashboard/feeds.tmpl]: Sounds great, thanks so much for the advice!

Comment (CaiCandong) [reply templates/user/dashboard/feeds.tmpl]: Done [`17fea19` (#26956)](https://github.com/go-gitea/gitea/pull/26956/commits/17fea1955c935800306ad6b6b177a4c7757a2218)

**Review by wxiaoguang — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by silverwind** [comment]
Dashboard fails to load for me with 500 on my local instance after this:

```
Render failed, failed to render template: user/dashboard/dashboard, error: template error: builtin(static):user/dashboard/feeds:14:20 : executing "user/dashboard/feeds" at <.GetOpType.InActions>: can't evaluate field InActions in type activities.ActionType
----------------------------------------------------------------------
					{{if .GetOpType.InActions "create_repo"}}
					               ^
```

**Comment by CaiCandong** [comment]
Let me check It.

**Comment by silverwind** [comment]
Yes, I'm at https://github.com/go-gitea/gitea/commit/bafe1334c6e46c0a0398c645da97b53f42996ab8.

**Comment by CaiCandong** [comment]
> Yes, I'm at [bafe133](https://github.com/go-gitea/gitea/commit/bafe1334c6e46c0a0398c645da97b53f42996ab8).

Thanks, indeed this function was missing.

**Comment by silverwind** [comment]
Must be something else. I defintely got https://github.com/go-gitea/gitea/commit/30cea70dd8e0a4583d8f63795b21551ce44ce223 on my `main` branch that exactly matches the remote.

**Comment by CaiCandong** [comment]
https://github.com/go-gitea/gitea/blob/bafe1334c6e46c0a0398c645da97b53f42996ab8/models/activities/action.go#L129-L136
Does this code exist locally? It's kind of weird.

**Comment by silverwind** [comment]
Yes, it's there. Maybe I will dive deeper later, unless you find it first. You should be able to reproduce this hopefully. Or maybe it's something weird about my database.

**Comment by wxiaoguang** [comment]
It can't be related to database.

Have you re-compiled and restarted the Gitea instance? Or could you try to do it manually in case it's the `air` bug?

**Comment by silverwind** [comment]
I ran `make clean-all build` and that seems to have fixed it. Bad cache somewhere I assume.

**Pull request merged**